docs/source/release_notes.rst

```diff
@@ -69,6 +70,8 @@ Breaking Changes
 * ``last_time_index``, ``secondary_time_index``, and ``interesting_values`` are no longer attributes
   of an entityset’s tables that can be accessed directly. Now they must be accessed through the metadata
   of the Woodwork DataFrame, which is a dictionary.
+* The helper function ``list_variable_types`` has been replaced with ``list_logical_types``.
+  To help minimize breaking changes, ``list_variable_types`` will return the same output as ``list_logical_types``.
```

Comment:
I think I'd tweak this wording a little bit. The first sentence says we replaced a function, but the second indicates the function still exists. Maybe we could say something like this:

The helper function `list_variable_types` will be removed in a future release and replaced by `list_logical_types`. In the meantime, `list_variable_types` will return the same output as `list_logical_types`.

Reply:
fixed